cypress/support/step-definitions/accordion-steps.js

```diff
@@ -43,6 +44,10 @@ When('I expand accordionRow via click', () => {
   accordionTitleContainer().click();
 });
 
+When('I expand accordionRow via click in no iFrame', () => {
```

Comment:
Do we need this step ? It isn't used anywhere

Reply:
deleted